.github/workflows/build-and-unit-tests.yaml

```diff
@@ -12,7 +12,7 @@ on:
       timeout:
         required: false
         type: number
-        default: 25
+        default: 45
```

Comment:
That's a big jump in time for this workflow. Is it necessary that this is part of post-commit?

Reply:
@ubcheema can this be one of the frequent pipelines / perf regression?

Reply:
I need to review why the run time has increased so much. Apparently. we are only adding some perf tests.

Reply:
Any update?

Reply:
Agreed. It is difficult identify the cause of ci timeout in this PR last week.

Sounds good. I removed the tunneler test from this PR.